src/main/kotlin/org/rust/ide/typing/paste/RsImportCopyPasteProcessor.kt

```diff
@@ -21,10 +21,17 @@ import com.intellij.psi.PsiFile
 import com.intellij.psi.impl.source.tree.injected.changesHandler.range
 import org.rust.ide.inspections.fixes.QualifyPathFix
 import org.rust.ide.inspections.import.AutoImportFix
+import org.rust.ide.refactoring.move.common.RsMoveUtil.containingModOrSelf
```

Comment:
nit: It would be nice not to have dependency on `org.rust.ide.refactoring.move` package here. Maybe it's better to extract `containingModOrSelf` to better place?